pkg/inventory/inventory-client.go

```diff
@@ -397,35 +397,9 @@ func (cic *ClusterClient) ListClusterInventoryObjs(ctx context.Context) (map[str
 	return identifiers, nil
 }
 
-// createInventoryObj creates the passed inventory object on the APIServer.
-func (cic *ClusterClient) createInventoryObj(obj *unstructured.Unstructured, dryRun common.DryRunStrategy) (*unstructured.Unstructured, error) {
```

Comment:
is this unused or moved?

Reply:
Since this is such a big PR, maybe move this change out by itself.

Reply:
It was unused.

Reply:
https://github.com/kubernetes-sigs/cli-utils/pull/657